plugins/sdm/config_template.py

```diff
@@ -26,7 +26,8 @@
     'ENABLE_RESOURCES_FUZZY_MATCHING': str(os.getenv("SDM_ENABLE_RESOURCES_FUZZY_MATCHING", 'true')).lower() == 'true',
     'EMAIL_SLACK_FIELD': os.getenv("SDM_EMAIL_SLACK_FIELD"),
     'EMAIL_SUBADDRESS': os.getenv("SDM_EMAIL_SUBADDRESS"),
-    'GROUPS_TAG': os.getenv("SDM_GROUPS_TAG")
+    'GROUPS_TAG': os.getenv("SDM_GROUPS_TAG"),
+    'REQUIRED_FLAGS': os.getenv("SDM_REQUIRED_FLAGS", "")
```

Comment:
Just one silly request: please make this None by default to keep it consistent with the rest of variables.